src/api/controllers/quotesController.js

```diff
@@ -33,6 +36,7 @@ const quoteController = async (req, res, next) => {
     let font = fonts[req.query.font] ? fonts[req.query.font] : fonts['default'];
 
     let quoteType = req.query.quoteType || '';
+    let unsplashQuery = req.query.unsplashQuery || '';
```

Comment:
NOTE: If multiple unsplashQuery params are given 
(e.g. ?unsplashQuery=morning&unsplashQuery=night),
this may arrive as an array instead of a single string.